performance_demo/index.html

```diff
@@ -86,6 +86,17 @@ <h2>Load resources</h2>
               id="type-both" />
           <label for="type-both">Both</label><br>
         </fieldset>
+        <fieldset>
+          <input type="checkbox" name="clear" value="cookies"
+              id="clear-cookies" checked />
+          <label for="clear-cookies">Cookies</label><br>
```

Comment:
Nit for a future patch: You shouldn't need line-breaks here. You can do all the styling you need via CSS. :)